### PR TITLE
blockdev_full_backup:update auto-finalize to true in manual_completed

### PR DESCRIPTION
**Pull request by aliang123**
Set auto-finalize to true in manual_completed to cover libvirt scenario

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2121351

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M.q35: PASS (77.67 s)

**Comment by aliang123** [comment]
@zhencliu @MiriamDeng, pelase help to review, thanks.

File: qemu/tests/cfg/blockdev_full_backup.cfg [hunk 1]
```diff
@@ -77,7 +77,7 @@
                     auto-finalize = true
                 - manual_completed:
                     auto-dismiss = false
-                    auto-finalize = false
+                    auto-finalize = true
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_full_backup.cfg:80]: hi @aliang123 , it looks that the test scenario changes if we set auto-finalize to true, the 'auto-finalize=false' test scenario was gone, is that OK?

Comment (zhencliu) [reply qemu/tests/cfg/blockdev_full_backup.cfg:80]: talked with aihua, backup will not cover auto-finalize=false, but other block jobs will cover this.

**Comment by zhencliu** [comment]
LGTM

**Comment by MiriamDeng** [comment]
LGTM
 (1/2) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M.with_data_plane: STARTED
 (1/2) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M.with_data_plane: PASS (205.58 s)
 (2/2) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M: STARTED
 (2/2) Host_RHEL.m8.u7.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.7.0.ppc64le.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M: PASS (183.61 s)

**Pull request merged**